### PR TITLE
Add create, mint, transfer, search method for native nft

### PR DESCRIPTION
**Pull request by woomurf**
Add methods for native nft
- createNftCollection
- mintNft
- transferNft
- searchNft

### CODE REVIEW

**Review by yoojinko — APPROVED** [review]
LGTM, Thanks!

File: src/types.ts [hunk 1]
```diff
@@ -749,3 +749,41 @@ export interface DepositHistory {
...
+
+export interface TransferNftParams {
+  address: string;
+  chain: string;
+  network: string;
+  appId: string;
+  collectionId: string;
+  tokenId: number;
+  toAddress: string;
+}
```
Comment (ehgmsdk20) [inline src/types.ts:789]: Add newline?

**Review by platfowner — APPROVED** [review]
LGTM
Thanks! 👍

File: src/types.ts [hunk 2]
```diff
@@ -749,3 +749,41 @@ export interface DepositHistory {
...
+  chain: string;
+  network: string;
+  appId: string;
+  collectionId: string;
+  metadata: object;
+  toAddress: string;
+  tokenId?: number;
+}
+
+export interface NftSearchOption {
```
Comment (jiyoung-an) [inline src/types.ts:775]: minor) How about `SearchNftOption`?

**Review by jiyoung-an — APPROVED** [review]
LGTM, thanks!

File: src/types.ts (hunk 1 above)
Comment (woomurf) [reply src/types.ts]: done!

File: src/types.ts (hunk 2 above)
Comment (woomurf) [reply src/types.ts]: done!

**Comment by woomurf** [comment]
TrailingUrl has been updated according to the changed route path.

File: src/types.ts (hunk 1 above)
Comment (woomurf) [reply src/types.ts]: PTAL~ @ehgmsdk20

**Review by ehgmsdk20 — APPROVED** [review]
LGTM, thanks!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**